mne/viz/utils.py

```diff
@@ -1536,6 +1536,45 @@ def _setup_cmap(cmap, n_axes=1, norm=False):
     return cmap
 
 
+def _format_ch_names(ch_names, n_ch=6):
+    """Format channel names into a string."""
+    ch_string = ", ".join(ch_names[:n_ch])
+    if len(ch_names) > n_ch:
+        warn('More than %d channels, truncating title ...' % n_ch)
+        ch_string += ", ..."
+    return ch_string
+
+
+def _prepare_joint_axes(n_maps, figsize=None):
+    """Prepare axes for topomaps and colorbar in joint plot figure.
+
+    Parameters
+    ----------
+    n_maps: int
+        Number of topomaps to include in the figure
+    figsize: tuple
+        Figure size, see plt.figsize
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        Figure with initialized axes
+    main_ax: matplotlib.axes._subplots.AxesSubplot
+        Axes in which to put the main plot
+    map_ax: list
+        List of axes for each topomap
+    cbar_ax: matplotlib.axes._subplots.AxesSubplot
+        Axes for colorbar next to topomaps
+    """
+    import matplotlib.pyplot as plt
+    fig = plt.figure(figsize=figsize)
+    main_ax = fig.add_subplot(212)
+    ts = n_maps + 2
+    map_ax = [plt.subplot(4, ts, x + 2 + ts) for x in range(ts - 2)]
+    cbar_ax = plt.subplot(4, 3 * (ts + 1), 6 * (ts + 1))
```

Comment:
... maybe we should also document what the magic numbers in these numbers are *for*. Which I guess I should have done in the first place ...